pkg/provisioner/ironic/ironic.go

```diff
@@ -726,7 +771,7 @@ func (p *ironicProvisioner) getUpdateOptsForNode(ironicNode *nodes.Node, checksu
 	if _, ok := ironicNode.InstanceInfo["root_gb"]; !ok {
 		op = nodes.AddOp
 		p.log.Info("adding root_gb")
-	} else if ironicNode.InstanceInfo["root_gb"] != 10 {
+	} else {
```

Comment:
I'm surprised this is even being set since we're working with whole disk images, root_gb is irrelevant. :\

Reply:
I don't know? The advice I had when originally working on the provisioning logic was that we had to have that value set to something. If we don't need it, we can deal with that in a separate PR.

Reply:
There are a couple different variables, but yeah.. :\

Reply:
Let's address this question separately.